lib/apiservers/engine/backends/cache/image_cache.go

```diff
@@ -181,7 +180,18 @@ func (ic *ICache) getImageByNamed(named reference.Named) (*metadata.ImageConfig,
 		}
 	}
 
-	return copyImageConfig(config), nil
+	return copyImageConfig(config)
+}
+
+// Add the "sha256:" prefix to the image ID if missing.
+// Don't assume the image id in image has "sha256:<id> as format.  We store it in
+// this format to make it easier to lookup by digest
+func prefixImageID(imageID string) string {
+	if strings.HasPrefix(imageID, "sha256:") {
+		return imageID
+	} else {
```

Comment:
I could, it just looks nicer for some reason. I was not using else before some moment of time, but last several years I tend to write like this.